spec/top_level_menu_spec.rb

```diff
@@ -22,7 +22,7 @@
     it 'verify that tshirts button is diplayed' do
       expect(tplevelmenu_tab.tshirts.displayed?).to be(true)
     end
-
+    g
```

Comment:
?

Reply:
Just occasionally wrong button pressed. Now it`s removed.